docs/fundamentals/flow/create-flow.md

```diff
@@ -186,6 +186,103 @@ t.start()  # start Flow in new Thread
 e.set()  # set event and stop (unblock) the Flow
 ```
 
+(restart-flow)=
+## Restart
+
+While developing your Flow, it can be useful to have the Flow be restarted automatically as you keep applying changes to the Flow YAML.
+
+For this you can use the `restart` argument for the Flow so that it watches changes in the configuration YAML code and makes sure that the Flow is restarted with the new configuration.
+
+````{admonition} Caution
+:class: caution
+This feature is thought to let the developer iterate faster while developing but is not intended to be used in production environment.
```

Comment:
See above comment and adjust accordingly

Reply:
What is exactly that I have to do?

Reply:
I think I rewrote the phrase "For this you can use the Flow's `restart` argument to restart the Flow with the updated configuration every time you change the YAML configuration." (or a very similar phrase) in prior commit suggestion.